ddtrace/writer.py

```diff
@@ -112,26 +113,50 @@ def _on_shutdown(self):
                     time.sleep(0.05)
 
     def _target(self):
+        last_error_ts = 0
```

Comment:
think about moving this to an attribute on AgentWriter so it's easier to test against `self.last_error_ts`